x/gov/types/expected_keepers.go

```diff
@@ -49,3 +49,17 @@ type BankKeeper interface {
 	SendCoinsFromAccountToModule(ctx sdk.Context, senderAddr sdk.AccAddress, recipientModule string, amt sdk.Coins) error
 	BurnCoins(ctx sdk.Context, name string, amt sdk.Coins) error
 }
+
+//_______________________________________________________________________________
+// Event Hooks
+// These can be utilized to communicate between a governance keeper and another
+// keepers.
+
+// GovHooks event hooks for governance proposal object (noalias)
+type GovHooks interface {
+	AfterProposalSubmission(ctx sdk.Context, proposalID uint64)                         // Must be called when proposal is submitted
+	AfterProposalDeposit(ctx sdk.Context, proposalID uint64, depositAmount sdk.Coins)                            // Must be called when deposit finish
+	AfterProposalVote(ctx sdk.Context, proposalID uint64, voterAddr sdk.AccAddress, option VoteOption)     // Must be called when vote on a proposal
```

Comment:
Should we pass along the vote option in the hook, or assume that if the hook recipient needs the vote option, it can get it from querying the govkeeper?  Like I think the point of hooks is to pass along minimal information that the recipient can use the query for more info if it needs.  @ValarDragon wdyt?

Reply:
That sounds fine to me, is there an existing efficient query to get the vote option? (efficient as in no iteration)

Reply:
Yeah, votes are keyed in the store by proposalID+voterAddress

Reply:
ok SGTM to remove the option here. (And ditto for depositAmount in AfterProposalDeposit)

Reply:
Oh good point.  But more importantly, the deposit hook, needs to include the depositer address.